runtime/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,156 @@
+    root: AccountId,
+    id: ParaId,
+) -> Value {
+    let config = RuntimeGenesisConfig {
+        balances: BalancesConfig {
+            balances: endowed_accounts
+                .iter()
+                .cloned()
+                .map(|k| (k, 1u128 << 60))
+                .collect::<Vec<_>>(),
+        },
+        parachain_info: ParachainInfoConfig {
+            parachain_id: id,
+            ..Default::default()
+        },
+        collator_selection: CollatorSelectionConfig {
+            invulnerables: invulnerables
+                .iter()
+                .cloned()
+                .map(|(acc, _)| acc)
+                .collect::<Vec<_>>(),
+            candidacy_bond: EXISTENTIAL_DEPOSIT * 16,
+            ..Default::default()
+        },
+        session: SessionConfig {
+            keys: invulnerables
+                .into_iter()
+                .map(|(acc, aura)| {
+                    (
+                        acc.clone(),                 // account id
+                        acc,                         // validator id
+                        template_session_keys(aura), // session keys
+                    )
+                })
+                .collect::<Vec<_>>(),
+            ..Default::default()
+        },
+        polkadot_xcm: PolkadotXcmConfig {
+            safe_xcm_version: Some(SAFE_XCM_VERSION),
+            ..Default::default()
+        },
+        sudo: SudoConfig { key: Some(root) },
+        ..Default::default()
+    };
+    serde_json::to_value(config).expect("Could not build genesis config.")
+}
+
+fn local_testnet_genesis() -> Value {
+    testnet_genesis(
+        // initial collators.
+        vec![
+            (
+                get_account_id_from_seed::<sr25519::Public>("Alice"),
+                get_collator_keys_from_seed::<AuraId>("Alice"),
+            ),
+            (
+                get_account_id_from_seed::<sr25519::Public>("Bob"),
+                get_collator_keys_from_seed::<AuraId>("Bob"),
+            ),
+        ],
+        vec![
+            get_account_id_from_seed::<sr25519::Public>("Alice"),
+            get_account_id_from_seed::<sr25519::Public>("Bob"),
+            get_account_id_from_seed::<sr25519::Public>("Charlie"),
+            get_account_id_from_seed::<sr25519::Public>("Dave"),
+            get_account_id_from_seed::<sr25519::Public>("Eve"),
+            get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+            get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+            get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+            get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+            get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+            get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+            get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+        ],
+        get_account_id_from_seed::<sr25519::Public>("Alice"),
+        1000.into(),
```

Comment:
Is just a minor thing, but we had 2000 before no? https://github.com/r0gue-io/base-parachain/pull/52/files#diff-f2256017c4e606a30f53708578da46675512cc1300be6542ed5f77e58af0d6dcL110

Reply:
Resolved